CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing to the RGSoC Teams App
+
+* Introduction, thanks for contributing
+* [Code of Conduct](https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/master/CODE_OF_CONDUCT.md)
+
+## Reporting a bug
+
+* Very valuable
+* Check if issue already exists
+
+## Suggesting a new feature
+
+### Labels
+
+* Dibs
+* Ping via comment and/or join Slack
+
+## Creating a Pull Request
+(inspired by [jbuilder](https://github.com/rails/jbuilder/blob/master/CONTRIBUTING.md))
+
+
+* If related to an issue: better ask first
+* Fork repo
+* Install locally (see [README](https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/master/README.md) for hints)
+* Make sure tests pass locally: `rake spec`
+
+### Create a Topic Branch
+
+Make sure your fork is up-to-date and create a topic branch for your feature or bug fix.
+
+```
+git checkout master
+git pull upstream master
+git checkout -b my-feature-branch
+```
+
+### Write Code
+
+* [Ruby Style Guide](https://github.com/bbatsov/ruby-style-guide)
+* With tests (TODO: can be hard for beginners)
+* Commits w/ descript
+
+### Make a Pull Request
+
+* Point to GH docs?
+
+### Be Patient
+
+
+## No idea where to put this:
+
+* Purely code-cosmetic contributions are likely to not be accepted
```

Comment:
Yes, something like that. Or the style of writing specs. Or line-breaks.

I'm thinking this only creates noise and is somewhat subject to personal taste.